.ddev/config.yaml

```diff
@@ -3,8 +3,8 @@ type: drupal9
 docroot: web
 php_version: "8.1"
 webserver_type: nginx-fpm
-router_http_port: "80"
-router_https_port: "443"
+router_http_port: "9082"
+router_https_port: "9443"
```

Comment:
No! I think you could do this as a separate gitignored file in your own instance? Looks like something you can put into `config.local.yaml` and you won't even have to worry about adding it to `.gitignore`.